tools/coqdep_common.ml

```diff
@@ -20,14 +20,15 @@ open Minisys
     options (see for instance [option_dynlink] below).
 *)
 
+let coqdep_warning args =
+  eprintf "*** Warning: @[";
+  kfprintf (fun fmt -> fprintf fmt "@]\n%!") err_formatter args
+
```

Comment:
I don't remember whether the proper hooks were plugged before this patch, but it would be nice to have terminal colouring in that function. I remember that @SkySkimmer added lightweight support for colour in `configure`, the same thing should be done here.

Reply:
That never had color except for a couple of cases where it was calling Coq's feedback, but IMHO it should use that. Note that this component is critical dependency-wise as it has to be bootstrapped as soon as possible. Ideally that just depend on `coqfind`.